runtime/compiler/control/MethodToBeCompiled.hpp

```diff
@@ -119,15 +126,18 @@ struct TR_MethodToBeCompiled
                                        // The flag in methodInfo is not enough because it may indicate true when
                                        // the entry is queued, but change afterwards if method receives samples
                                        // to be upgraded to hot or scorching
+   int16_t                _index;
    uint8_t                _freeTag; // temporary to catch a nasty bug
    uint8_t                _weight; // Up to 256 levels of weight
    bool                   _hasIncrementedNumCompThreadsCompilingHotterMethods;
    uint8_t                _jitStateWhenQueued;
-   bool                   _remoteCompReq; // comp request should be sent remotely to JITaaS server
-   JITServer::ServerStream  *_stream; // a non-NULL field denotes an out-of-process compilation request
+#if defined(JITSERVER_SUPPORT)
+   bool                   _remoteCompReq; // Comp request should be sent remotely to JITServer
+   JITServer::ServerStream  *_stream; // A non-NULL field denotes an out-of-process compilation request
    char                  *_clientOptions;
-   size_t                _clientOptionsSize;
-   TR_Hotness            _origOptLevel; // used to cache original optLevel when transforming a remote sync compilation to a local cheap one
+   size_t                 _clientOptionsSize;
```

Comment:
In a future PR we may want to move `clientOptions` from `TR_MethodToBeCompiled` to `CompilationInfoPerThreadRemote`